service/src/main/java/bio/terra/cbas/runsets/outputs/OutputGenerator.java

```diff
@@ -17,6 +17,7 @@
 
 public class OutputGenerator {
 
+  @SuppressWarnings("unchecked")
```

Comment:
Added to suppress Unchecked cast: 'java.lang.Object' to 'java.util.Map<java.lang.String,java.lang.Object>'